tutorials/lbs/first/first_example.lua

```diff
@@ -112,15 +112,15 @@ materials[1]:SetTransportXSections(xs_matA)
 
 --[[ @doc
 ## Volumetric Source
-We create a lua table containing the volumetric multigroup source and assign it to the material by passing that array.
+We create a volumetric multigroup source which will be assigned to the material with given block IDs.
+Volumetric sources are assigned to the solver via the `options` parameter in the LBS block (see below).
```

Comment:
so, does this mean I can have several sources, as in `volumetric_sources = { mg_src1, mg_src2 }` ? If so, that's great!

Reply:
> does this mean I can have several sources, as in `volumetric_sources = { mg_src1, mg_src2 }` ?

Yes.